first-react-app/src/api/lastFMAPI.ts

```diff
@@ -0,0 +1,114 @@
+import fetch from "cross-fetch";
+
+interface getArtistParams {
```

Comment:
you should use `UpperCamelCase` for interface name.

https://google.github.io/styleguide/tsguide.html#identifiers

```suggestion
interface GetArtistParams {
```